testfixtures/logcapture.py

```diff
@@ -120,12 +181,12 @@ def uninstall(self):
                 logger.setLevel(self.old['levels'][name])
                 logger.handlers = self.old['handlers'][name]
                 logger.disabled = self.old['disabled'][name]
-                logger.propagate = self.old['progagate'][name]
+                logger.propagate = self.old['propagate'][name]
             self.instances.remove(self)
 
     @classmethod
     def uninstall_all(cls):
-        "This will uninstall all existing :class:`LogHandler` objects."
+        """This will uninstall all existing :class:`LogHandler` objects."""
```

Comment:
Not sure changes like this are worth it, but if you must, please put them in their own commit.